guides/scripts/generate-hammer-reference.sh

```diff
@@ -200,7 +206,7 @@ cat "$SRC" | while read line ; do
         "Usage:")
             [ -n "$DEBUG" ] && echo -e "I: [Usage]ID: $id_core"
             [ -n "$NOOP" ] || [ -n "$ASSEMBLY_ONLY" ] || \
-                echo -e ".Usage" >>$mod_file
+                echo -e "*Usage*\n" >>$mod_file
```

Comment:
Alternative: `Usage::` but I have not seen the output yet, so please take this comment with a grain of Salt.

Reply:
@apinnick suggested the same, but I hesitate using it.

The definition list doesn't make a good replacement for "informal heading" or "block title". It's supposed to define terms, formally speaking.
The blocks below a definition list item need to be connected with `+` for proper indentation in the output. This situation doesn't seem to happen in the reference at the moment. But if it did, I'm not sure the script could insert `+` at the right places with its current logic.

So technically, there's nothing stopping us from using it.